doc/tech/glossary.md

```diff
@@ -146,15 +146,24 @@ Terms not defined by Kubernetes:
 - Observability = Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. ([Wikipedia:Observability](https://en.wikipedia.org/wiki/Observability))
 
 <a name="1.7"></a>
-## 1.7 Other Referenced Terminology
+## 1.7 Test Related Terminology
+
+- **Performance Measurement:** The procedure or set of operations having the object of determining a Measured Value or Measurement Result. In this context, Performance Measurements reflect data generated and collected within the cloud infrastructure, that reflects a performance aspect of the cloud infrastructure. For example, a count of frames or packets traversing an interface, memory usage information, other resource usage and availability, etc. These data may be instantaneous or accumulated, and made available (i.e. exposed) based on permissions and contexts (e.g., workload vs. infra).
+
+- **Performance Testing**: The main objective of CNTT performance testing is to understand if a potential BOM is functioning correctly to get expected performance. It requires a set of performance testing tools (open source) that help with the dimensioning of a solution by an operator. Testing results may be useful to compare infrastructure capabilites between the System Under Test (SUT) and a CNTT reference implementation of RA-2. Performance testing for the purpose of comparing between different commercial implementations is not a goal of CNTT. Performance testing relies on well established benchmark specifications.
```

Comment:
This section should become an introduction for the Test Related Terminology Section, so I suggest to delete the text here and move under the section title.
```suggestion
    
```

Reply:
Updated per CVC

Reply:
@acmacm can this be resolved?